src/graphics/program-lib/chunks/reflectionPrefilteredCubeLod.frag

```diff
@@ -5,7 +5,9 @@
 uniform samplerCube texture_prefilteredCubeMap128;
 #endif
 uniform float material_reflectivity;
-
+#ifdef CLEARCOAT
+    uniform float material_clear_coat_reflectivity;
+#endif
```

Comment:
Would be nice if this was consistent everywhere: calcReflection() function used by the two invocations.

Reply:
made it consistent - but didn't separate out into chunks